imports/requestAudioBank/client.lua

```diff
@@ -0,0 +1,15 @@
+---Loads audio bank 
+---@param audioBank string
+---@param timeout number?
+---@return boolean
+function lib.requestAudioBank(audioBank, timeout)
```

Comment:
@Ehbw Can I get you to update `requestAudioBank` to be in-line with the changes in https://github.com/overextended/ox_lib/commit/0f0c7a659124659b038dd2e0be9f004b76c7ce10

Also needs a TypeScript counterpart.